ops/travis/deploy/master

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+# pull the current tested build
+export DOCKER_IMAGE_TAG=$(exec ops/travis/helpers/build_docker_image_tag)
+make pull
+
+# show current images on system
+docker images
+
+# these variable must be available securely from travis
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+
+# re-tag build to master-latest
```

Comment:
FYI: [other options to pass info to to makefiles](https://stackoverflow.com/questions/2826029/passing-additional-variables-from-command-line-to-make)

Reply:
I am using the option that the travis guy [recommend](https://docs.travis-ci.com/user/docker/#pushing-a-docker-image-to-a-registry)
and actually you probably do not want to put the dockerhub credentials anywhere visible

Reply:
make sense .. it was just for info